hollow/src/main/java/com/netflix/hollow/api/client/HollowClientUpdater.java

```diff
@@ -83,7 +83,7 @@ public HollowClientUpdater(HollowConsumer.BlobRetriever transitionCreator,
     /**
      * Updates the state to the provided version
      *
-     * @param version the version to update to
+     * @param requestedVersion the version to update to
```

Comment:
I suggest that JavaDoc is updated to specify the behaviour in more detail and the error conditions with `@throws IllegalArgumentException`.